static/js/components/LearnerSearch.js

```diff
@@ -232,7 +238,7 @@ export default class LearnerSearch extends SearchkitComponent {
             orderKey="_term"
             id="payment_status"
           />
-        </FilterVisibilityToggle>
+        </FilterVisibilityToggle> : null }
         <FilterVisibilityToggle
```

Comment:
It looks like `program.enrollments` is not available to learners, would they be able to see this filter at all?